.pre-commit-config.yaml

```
@@ -0,0 +1,21 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        language_version: python3
+        args: []
+        exclude: gen/
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.971
+    hooks:
+      - id: mypy
+#This can potentially be a slow pre-commit hook.
+  - repo: local
+    hooks:
+      - id: pytest-check
+        name: pytest-check
+        entry: pytest 
+        language: system
+        pass_filenames: false
```

Should this also be like the "robust" pytest pre-commit seen in ISAR?
```suggestion
       entry: venv/bin/pytest 
        language: script
        pass_filenames: false
```